query/query.go

```diff
@@ -111,51 +110,76 @@ type Latency struct {
 	Json       time.Duration `json:"json_conversion"`
 }
 
+// params contains the list of parameters required to execute a query.
 type params struct {
-	Alias      string
-	Type       string
-	Count      int
-	Offset     int
-	AfterUID   uint64
-	DoCount    bool
-	GetUid     bool
-	Order      []*pb.Order
-	Var        string
-	NeedsVar   []gql.VarContext
+	Alias    string
+	Count    int         // Value of first parameter.
```

Comment:
The word first could be confusing. I initially thought there was a sequence of parameters somewhere, and first meant the first of those parameters. Could quote it may be.